### ISSUE TITLE
Screen reader does not convey any confirmation message after applying 'Viewport selection'.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#122]
User Experience:

If user is not notified by AT, then it will be hard to understand for visually impaired user that what UI changes happened on the page.

 

Re-pro steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode

4. Navigate to the Bottom nav and select Viewport Selection button.

5. Enable screen reader and select many option Extra-large screen, Zoom out, large screen, medium screen, small screen, extra small screen.

6. Verify the issue.

 

Actual Result:

Screen reader does not convey any confirmation message after applying ‘Extra-large screen, Zoom out, large screen, medium screen, small screen, extra small screen’.

 

Expected Result:

Screen reader should provide a confirmation message about viewport selection updates.

 

Note:

1. Similar issue exist with Chrome + JAWS and Chrome + JAWS

2. Similar issue exist with controls: Undo, Redo

3. Similar issue exist throughout the application, after applying any update , screen reader does not provide any information.

 

MAS Reference:

MAS 1.3.1 - Info and Relationships (40)

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries.
